CHANGELOG.md

```diff
@@ -4,3 +4,4 @@
 
 * New app created with stripes-cli
 * Add View MARC authority record permission. (UIMARCAUTH-6)
+* Add Edit MARC authority record permission. (UIMARCAUTH-5)
```

Comment:
@BogdanDenis, what do you think about using `ui-quick-marc` changelog template for this module? I think the links in the changelog might be handy.

Reply:
@Vladyslav-Velytskyi good idea, I'll update changelog